packages/gatsby-plugin-google-analytics/src/index.js

```diff
@@ -62,6 +64,9 @@ function OutboundLink(props) {
 OutboundLink.propTypes = {
   href: PropTypes.string,
   target: PropTypes.string,
+  eventCategory: PropTypes.string,
+  eventAction: PropTypes.string,
+  eventLabel: PropTypes.string,
```

Comment:
Should we add `eventValue` prop here as well?

Reply:
Don't send the `eventValue` to Google if it's not set. It's an optional setting.

Reply:
I was wondering about the same thing, but then I saw that there is similar code in the `trackCustomEvent` function here https://github.com/gatsbyjs/gatsby/blob/93f0645dc0088ef114eb6ca7ed05f23cae3556ce/packages/gatsby-plugin-google-analytics/src/index.js#L74-L92

So that's why I decided to lose all checks and just send the `eventValue`. Are you OK with that or you prefer me to change it?

Reply:
I think we should at least add `eventValue` to props. As for the `undefined` value - ga should handle it properly (even `JSON.stringify()` would simply remove it from the output)